scanpy/tools/_tsne.py

```diff
@@ -76,8 +76,11 @@ def tsne(
     -------
     Returns `None` if `copy=False`, else returns an `AnnData` object. Sets the following field:
 
-    `adata.obsm['X_tsne']` : :class:`numpy.ndarray` (dtype `float`)
+    `adata.obsm['X_tsne']` : :class:`numpy.ndarray` | :class: `` (dtype `float`)
```

Comment:
seems like you missed something here, otherwise beautiful!

Reply:
Yes, think this should be gone after [303c385](https://github.com/scverse/scanpy/pull/2742/commits/303c385077d67b88440114684c6dc212b834f31d).